system/classes/Kohana/I18n.php

```diff
@@ -124,43 +141,31 @@ public static function load($lang)
 				// files from overloading more specific files
 				$table += $t;
 			}
-
-			// Remove the last part
-			array_pop($parts);
 		}
-		while ($parts);
+		while (array_pop($parts));
 
 		// Cache the translation table locally
 		return I18n::$_cache[$lang] = $table;
 	}
-
 }
 
 if ( ! function_exists('__'))
 {
 	/**
-	 * Kohana translation/internationalization function. The PHP function
-	 * [strtr](http://php.net/strtr) is used for replacing parameters.
+	 * Kohana translation/internationalization function.
 	 *
-	 *    __('Welcome back, :user', array(':user' => $username));
+	 *    $str = __('Welcome back, :user', [':user' => $username]);
 	 *
 	 * [!!] The target language is defined by [I18n::$lang].
 	 *
 	 * @uses    I18n::get
 	 * @param   string  $string text to translate
 	 * @param   array   $values values to replace in the translated text
-	 * @param   string  $lang   source language
+	 * @param   string  $lang   target language
 	 * @return  string
 	 */
-	function __($string, array $values = NULL, $lang = 'en-us')
+	function __($string, array $values = NULL, $lang = NULL)
```

Comment:
Why did you remove default lang?

Reply:
i dont do this https://github.com/koseven/koseven/pull/292/commits/b281407660117983edbb8bec6e7fe7f614653290#diff-83f0de0a4344b84921a83df373244907R87

Reply:
@piotrbaczek think, what you need to do to change default language before and now? :)